husarion_ugv_docking/launch/station.launch.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Husarion sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import imageio
+import yaml
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, OpaqueFunction
+from launch.substitutions import (
+    Command,
+    FindExecutable,
+    LaunchConfiguration,
+    PathJoinSubstitution,
+)
+from launch_ros.actions import Node
+from launch_ros.parameter_descriptions import ParameterValue
+from launch_ros.substitutions import FindPackageShare
+
+
+def generate_apriltag_and_get_path(tag_id):
+    from moms_apriltag import TagGenerator2
+
+    tag_generator = TagGenerator2("tag36h11")
+    tag_image = tag_generator.generate(tag_id, scale=1000)
+    with open(f"/config/apriltags/tag_{tag_id}.png", "w") as temp_file:
+        imageio.imwrite(temp_file.name, tag_image)
```

Comment:
I propose create appropriate folder if do not exist and add `apriltags` folder to `.gitignore` to do not push it unintentionally.
```suggestion
    os.makedirs(folder_path, exist_ok=True)
    with open(f"/config/apriltags/tag_{tag_id}.png", "w") as temp_file:
        imageio.imwrite(temp_file.name, tag_image)
```